packages/wdio-browserstack-service/src/launcher.ts

```diff
@@ -872,4 +883,27 @@ export default class BrowserstackLauncherService implements Services.ServiceInst
         BStackLogger.logToFile(`If facing any issues, please contact BrowserStack support with the Build Run Id - ${uuid}`, 'info')
         return uuid
     }
+
+    _setAccessibilityTrueInCapabilities(capabilities?: Capabilities.TestrunnerCapabilities) {
+        if (capabilities) {
+            if (Array.isArray(capabilities)) {
```

Comment:
do we have a utility to update any capabilities handling both w3c and nonw3c. Please check once.